ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -1802,3 +1802,227 @@ export const testFlex = (): Promise<boolean> =>
       reject(error);
     }
   });
+
+export const testPadding = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      await whisper.create({
+        label: 'Padding Property Test',
+        onClose: () => {
+          console.debug('closed');
+        },
+        components: [
+          {
+            body: 'Compare the two sets of boxes below. Does the second group have padding?',
+            type: WhisperComponentType.Markdown,
+          },
+          {
+            type: whisper.WhisperComponentType.CollapseBox,
+            open: true,
+            children: [
+              {
+                type: whisper.WhisperComponentType.Box,
+                direction: whisper.Direction.Horizontal,
+                justifyContent: whisper.JustifyContent.SpaceAround,
+                onClick: () => console.log('click the box'),
+                children: [
+                  {
+                    type: whisper.WhisperComponentType.Button,
+                    label: 'test',
+                    layout: {
+                      flex: '1',
+                    },
+                    onClick: () => console.log('Click the Button'),
+                  },
+                  {
+                    type: whisper.WhisperComponentType.Icon,
+                    name: 'favorite',
+                    size: whisper.IconSize.Small,
+                    layout: {
+                      flex: '1',
```

Comment:
That might not be related to this task, but do you think that's a better idea to make flex an enum?

Reply:
Unfortunately, Flex can take any valid flex string. So things like "1 0 50%", or "2", or even "0 1".  This property is def for power users. I think we could consider some defaults / enums, but we weren't sure how to go about doing that (we being Rich, JB, and I). Open to ideas!